proposals/dao/feiRari.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from 'chai';
+import { utils } from 'ethers';
+import { keccak256 } from 'ethers/lib/utils';
+import { RunUpgradeFunc, SetupUpgradeFunc, TeardownUpgradeFunc, ValidateUpgradeFunc } from '../../types/types';
+import hre, { ethers } from 'hardhat';
+
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {}
+const run: RunUpgradeFunc = async (addresses, oldContracts, contracts, logging = false) => {}
+const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {};
+
+const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts) => {
+    const { autoRewardsDistributor, rewardsDistributorAdmin } = contracts;
+    const AUTO_REWARDS_DISTRIBUTOR_ROLE = await rewardsDistributorAdmin.AUTO_REWARDS_DISTRIBUTOR_ROLE();
+
+    /// check that the contracts were wired together properly
+    expect(await rewardsDistributorAdmin.hasRole(AUTO_REWARDS_DISTRIBUTOR_ROLE, autoRewardsDistributor.address)).to.be.true;
+    expect(await autoRewardsDistributor.rewardsDistributorAdmin()).to.be.equal(rewardsDistributorAdmin.address);
```

Comment:
add a check that the OA timelock `isContractAdmin()` on the RDA
also check the rewardsDistributorContract on RDA is the Delegator

check `getNewRewardSpeed()` is correct for ARD, should be around 24 TRIBE/block

Reply:
Added `isContractAdmin` check.

I need to add a check for the second one

Reply:
Last one can prob do without

Reply:
Added a check for it.